router/routes/courses.js

```diff
@@ -620,18 +622,7 @@ function  generateListings(data, user, cb) {
                                 '        </a>';
                         }
                         html += '</br>';
-
-                        acl.isAllowed(user, classid, 'Remove', function (err, res) {
-                            if (res) {
-                                // html +=
-                                //     '<a class="actionbtn rmbtn">' +
-                                //     '          <span class="glyphicon glyphicon-remove"></span> Remove\n' +
-                                //     '        </a>' +
-                                //     '</td>';
-                            }
-                            // html += '</tr>';
-                            fcb(null,html);
-                        });
+                        fcb(null, html);
```

Comment:
What was this code doing, and now what is it doing?

Reply:
Remove function was commented out for some reason, and you should have instructor permission to remove a course. 

In general, this function is called to render the course list. With different permissions, you render different buttons for each class.